tags/faq/lwjgl.ytag

```diff
@@ -2,6 +2,7 @@ type: text
 
 ---
 
-If you're getting `java.lang.UnsatisfiedLinkError: Failed to locate library: lwjgl.dll` when running your project:
-  1. delete `<project dir>/.gradle/loom-cache/launch.cfg`
-  2. regenerate your run configs
+If you're getting `java.lang.UnsatisfiedLinkError: Failed to locate library: lwjgl.dll/dylib/so` when running your project:
+  1. Delete `<project dir>/.gradle/loom-cache/launch.cfg`
```

Comment:
This isn’t needed, it’s regenerated each time and likely already has the correct content. The issue is when moving a work space the path in the run config is wrong.